doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -33,8 +35,37 @@ Samples and applications
 
 This section describes the changes related to samples and applications.
 
-Wi-Fi®
-------
+Applications using the :file:`dfu_application.zip` file
+-------------------------------------------------------
+
+.. toggle::
+
+   For all applications using the :file:`dfu_application.zip` file generated by the |NCS| build system:
+
+     * Make sure that your DFU host tools support the :file:`dfu_application.zip` file with the new format version (``1``).
+       If the tools do not support the new format version and you cannot update them, you can manually align the content of the zip archive generated with format version ``1`` to version ``0``:
+
+     * Build your application in the same configuration with the |NCS| v2.6 release to obtain a reference file :file:`dfu_application.zip` with format version ``0``.
+     * Manually align the content of the :file:`dfu_application.zip` file generated with format version ``1``:
+
+       * Align the properties described in the :file:`manifest.json` file.
+         Make sure to update all of the fields that are used by the selected DFU host tool.
+       * Rename the binary files that are included in the zip archive to match the file names used by the updated manifest.
+         The binary file content is interoperable across |NCS| releases.
+
```

Comment:
Can we add a separate section to make clear that short-range Radio samples and application have been migrated to use IPC Radio. It's a bit of a repetition from the highlights section, but I'd like to make sure users are aware what changed underneath when they build radio samples. I propose something like this:

Samples and applications that use short-range radio
----------------------------------------------------------

All samples and application build for multi-core SoCs were migrated to use :ref:`ipc_radio` as the default image for the network core (on nRF5340) or the radio core (on nRF54H20).
The samples previously used for building firmware images for those cores are no longer supported: :ref:`zephyr:bluetooth-hci-ipc-sample`, :ref:`zephyr:nrf-ieee802154-rpmsg-sample` and :ref:`multiprotocol-rpmsg-sample`.

Reply:
I don't think 'no longer supported' is a good term. Maybe 'no longer used' would be better. Also there is one more sample :ref:`ble_rpc_host`.
On a sidenote: the use of the 4 samples is still possible, just we are not doing it.

Reply:
Correct, I'll update my comments.